modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
+
+Example: Imagine you're using a chat application and want to send a sharing link. With the Embed Mode, you can open the Web UI in a file picker-like interface, select files from Infinite Scale, and share them without ever having to leave the chat tab.
+
+The Embed Mode streamlines workflows and eliminates the need to switch between different applications to manage files.
+
+[discrete]
+=== Focused "New" menu
+
+The "New" menu items have been reorganized with the most frequently used items placed at the top. This rearrangement is based on user feedback, ensuring that the most important filetypes are readily accessible. In our commitment to Open Source and universal accessibility, we've revised the wording for Document labels to be more vendor-neutral. This change reflects our dedication to providing a user-friendly interface that caters to a universal user base.
+
+[discrete]
+=== 400% Faster Upload Preparation Time
+
+In the latest update, we've implemented significant improvements in the efficiency of folder tree creation during file uploads. By optimizing the process to run asynchronously and reducing the number of PROPFIND requests on nested folders, we've managed to substantially speed up this operation. In a test scenario with a folder containing 155 subfolders, the time to create these folders has been reduced from 20-30 seconds to just 5-7 seconds. While this duration may still be noticeable, it represents a considerable improvement, especially considering the limitations of client-side operations.
+
+Additionally, we've massively enhanced the upload preparation time. Rather than setting file data individually for each file, we now collect all necessary data first and then apply it in a single batch using Uppy's setState method. This approach streamlines the upload process, making it much quicker and more efficient for users
+
+[discrete]
+=== Default link permission
+
+With this new capability we've introduced a new configuration option that allows administrators to set the default permissions for sharing links. This feature is tailored to meet the varying security needs of different organizations.
+
+As an administrator, you now have the flexibility to define what the standard permissions for a link should be. This means you can choose whether to create a sharing link that is accessible to external parties with just a click, or opt for a more secure link that requires user authentication for access.
+
+The default link permissions is communicated through capabilities. The default setting for this config option is set to `default_link_permissions`:`1``, meaning that by default, anyone can access the link with viewer permissions. This enhancement is geared towards providing administrators with greater control over their organization's data sharing and security protocols.
```

Comment:
```suggestion
The default link permissions is communicated through capabilities. The default setting for this config option is set to `default_link_permissions`:`1`, meaning that by default, anyone can access the link with viewer permissions. This enhancement is geared towards providing administrators with greater control over their organizations data sharing and security protocols.
```

Reply:
The apostrophe needs to stay - the data belongs to the organization, so this is ownership "organisation's data"

Reply:
Whoops, thank you ☺️